concepts/Common/Data Analytics and BI/Embedded Dashboard/Embedded Dashboard.md

```diff
@@ -0,0 +1,32 @@
+If you need a reliable solution to monitor your Key Performance Indicators (KPIs) and track trends effortlessly, take a look at [DevExpress BI Dashboard](https://www.devexpress.com/products/net/dashboard/).
+
+![DevExpress BI Dashboard](/images/Common/Embedded_Dashboard.png)
+
+#### Discover Interactive Dashboards
+
+- Access a diverse range of data visualization components, including charts, cards, gauges, pivot grids, and more.
+
+- Use a drag-and-drop interface to customize your dashboard layout. BI Dashboard is built on DevExtreme UI components - you will not need to relearn.
```

Comment:
`you will not need to relearn` This is a strange statement. Do we need to add this?

Reply:
I really don't know how to formulate it another way. Maybe,
```suggestion
- Use a drag-and-drop interface to customize your dashboard layout. BI Dashboard is built on DevExtreme UI components - it will be easy to learn.
```